app/models/zip_export.rb

```diff
@@ -101,4 +101,9 @@ def zip!(input_dir, output_file)
       end
     end
   end
+
+  def generate_repositories_zip(tmp_dir, data, options = {})
```

Comment:
Try if you can remove options parameter altogether. I think it will crash, but I'm not sure, just try and if it doesn't crash, you can remove it.